### PR TITLE
[TASK] Display fees and load in RT (RT-3228)

### PR DESCRIPTION
**Pull request by mrajvanshy**

### CODE REVIEW

File: src/js/controllers/navbar.js [hunk 1]
```diff
@@ -31,12 +31,31 @@ module.controller('NavbarCtrl', ['$scope', '$element', '$compile', 'rpId',
...
   // Username
   $scope.$watch('userCredentials', function(){
     var username = $scope.userCredentials.username;
     if(username) $scope.loading = false;
     $scope.shortUsername = null;
+
+    if(!$scope.connected && username) {
+      $scope.message = 'Disconnected from the Ripple network'; 
+    }
+    else if ($scope.fee && $scope.fee > 0.012 && $scope.fee < (Options.max_tx_network_fee/1000000)) {
```
Comment (vhpoet) [inline src/js/controllers/navbar.js]: 0.012 should also be a config option

**Comment by rht** [comment]
@mrajvanshy you can avoid using the ```/1000000``` altogether

File: src/js/controllers/navbar.js [hunk 2]
```diff
@@ -31,12 +31,30 @@ module.controller('NavbarCtrl', ['$scope', '$element', '$compile', 'rpId',
...
+    else if ($scope.fee && $scope.fee > Options.low_load_threshold && $scope.fee < (Options.max_tx_network_fee/1000000)) {
+      $scope.serverLoad = 'mediumLoad';
+      $scope.message = 'Network fees are currently higher than normal. Fee: ' + $scope.fee + ' XRP';
+    } 
+    else if ($scope.fee && $scope.fee >= (Options.max_tx_network_fee/1000000)) {
+      $scope.serverLoad = 'highLoad';
+      $scope.message = 'Network fees are currently higher than your maximum. Fee: ' + $scope.fee + ' XRP';
+    }
+    else if ($scope.fee) {
+      $scope.message = 'Connected to the Ripple network. Fee: ' + $scope.fee + ' XRP';
```
Comment (rht) [inline src/js/controllers/navbar.js]: ```ripple.Amount.from_json($scope.fee).to_human()``` while removing the /1000000's from other places.

**Comment by rht** [comment]
Have been wondering if the server status popover should look like the ones in the bell/gear icon instead.

**Comment by mrajvanshy** [comment]
I don't think it should look like the drop downs from the bell/gear icons because this is just a popover that is providing information to the user. The user cannot click on the popover and be redirected to another page. Since it is simply information, I think it should look more like a popover than a clickable button.

**Comment by vhpoet** [comment]
LGTM

**Pull request merged**